### PR TITLE
css-positioning - Jay Ghil

### PR DESCRIPTION
**Pull request by jghil**
css-fixed-positioning
relative-positioning
absolute-positioning

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  top: 50%;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+  width: 300px;
+  height: 300px;
+
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:43]: you should have position absolute here as well.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
1 tiny update!

File: css-positioning/absolute.css (hunk 1 above)
Comment (jghil) [reply css-positioning/absolute.css:43]: done!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**